bindings/jupyter-modules/jupyter-ma-causal/src/components/multi-line-chart/index.js

```diff
@@ -0,0 +1,241 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {line as d3Line} from 'd3-shape';
+import {scaleLinear, scaleOrdinal} from 'd3-scale';
+import {schemeCategory10} from 'd3-scale-chromatic';
+import {extent as d3Extent} from 'd3-array';
+import {format as d3Format} from 'd3-format';
+
+const PADDING = {TOP: 10, BOTTOM: 20, LEFT: 20, RIGHT: 10};
+
+const getXScale = width =>
+  scaleLinear()
+    .domain([0, 1])
+    .range([PADDING.LEFT, width - PADDING.RIGHT]);
+
+const getYScale = height =>
+  scaleLinear()
+    .domain([0, 1])
+    .range([height - PADDING.BOTTOM, PADDING.TOP]);
+
+const colorScale = scaleOrdinal(schemeCategory10);
+
+export default class Chart extends Component {
+  static propTypes = {
+    data: PropTypes.object.isRequired,
```

Comment:
Do we want to reserve `data` from something else? It seems we are only destructing `lines` out of it in this `Chart` component.

Reply:
The data may contain other items besides `lines`, such as the name of the column. Because this is just a prototype right now, there's no way we can tell what the final input format looks like, we can leave it as `data` and it is easy enough to change later.